File: examples/swig/query.pl
#! /usr/bin/env perl

use jellyfish;

my $qf = jellyfish::QueryMerFile->new(shift @ARGV);
for my $s (@ARGV) {
  print($s, " ", $qf->get(jellyfish::MerDNA->new($s)), "\n");
}

File: examples/swig/query.rb
#! /usr/bin/env ruby

require 'jellyfish'

qf = Jellyfish::QueryMerFile.new(ARGV[0])
ARGV[1..-1].each { |s|
  print(s, " ", qf[Jellyfish::MerDNA.new(s)], "\n")
}

